tools/autograd/templates/Functions.cpp

```diff
@@ -676,6 +676,36 @@ Tensor var_backward(Tensor grad, const Tensor & self, IntArrayRef dim, bool unbi
   return (2.0 / (_safe_size(self.sizes(), dim) - unbiased)) * grad * (self - self.mean(dim, true));
 }
 
+Tensor var_std_mean_backward(const variable_list& grads, const Tensor & self, const Tensor & r1, const Tensor & r2, IntArrayRef dim, bool unbiased, bool keepdim, bool is_std) {
+  AT_ASSERTM(grads.size() > 0 && grads.size() <= 2, "grads size should be in interval [0,2]");
```

Comment:
shouldn't `grads.size()` always be 2?

Reply:
I think there should be a possibility to run backward to a specific result. For example just for mean or just for std.

Reply:
backwards functions aren't part of the public interface.